rest-service/manager_rest/resource_manager.py

```diff
@@ -177,17 +177,15 @@ def update_deployment_statuses(self, latest_execution):
         dep.latest_execution = latest_execution
         dep.deployment_status = dep.evaluate_deployment_status()
         self.sm.update(dep)
-        if dep.deployment_parents:
```

Comment:
I take it this doesn't need to happen because of some part of the previous PR I already forgot?

Reply:
it doesn't need to happen because of THIS pr, `graph.propagate_deployment_statuses(dep.id)` ~ `recalc_ancestors()`